components/file_attachment_list/file_attachment_list.jsx

```diff
@@ -79,6 +79,25 @@ export default class FileAttachmentList extends React.PureComponent {
             );
         }
 
+        if (fileInfos && fileInfos.length === 2) {
+            const fileType = getFileType(fileInfos[1].extension);
+
+            if (fileType === FileTypes.IMAGE || (fileType === FileTypes.SVG && enableSVGs)) {
+                return (
+                    <MultipleImageView
+                        fileInfo={fileInfos[1]}
```

Comment:
You want multiple files to be viewed directly, then why send only one file as the prop?